### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by fifilrv8** [osmandapp/OsmAnd#13553]
# 🚀 feature request

Hello,
I travel a lot offroad and I often use roadbooks. The information given by these RBs requires information on total km and partial km. It would be nice to have in Osmand a tripmaster widget representing 2 km displays with reset, 1 total and 1 partial (like on a car). The total would be reset by pressing the display and confirmation would be requested by pressing "yes" to authorize the reset. For the partial display, the reset would be done immediately after pressing the display. The km would be incremented on the 2 displays as soon as the vehicle is in motion. Obviously these displays would be active without being in navigation mode.
Please see if it is possible to integrate this in a future version. Of course, I remain available for any information.
cordially
best regards
[Roadbook.pdf](https://github.com/osmandapp/OsmAnd/files/7875478/Roadbook.pdf)

**Comment by scaidermern** [comment]
I think this is roughly the same request as #1734.

**Comment by fifilrv8** [comment]
Ce n'est pas tout a fait ce que je souhaite, pour moi il me faut juste un compteur Total et partiel avec Reset, mais sans avoir besoin de donner une destination. Ces compteurs doivent être incrémentés dés que le vehicule en en mouvement. Voir image jointe.
![trip-master](https://user-images.githubusercontent.com/97679187/149676889-5e43fa4b-647e-4caa-9d5a-18daa9c1badb.jpg)

**Comment by scaidermern** [comment]
I still think it is the same. I would expect the "trip computer" to work as soon as you move. Depending on the currently selected profile. Regardless of whether there is any active navigation.

**Comment by fifilrv8** [comment]
Yes that's exactly it, the 2 counters must start without a destination having been selected.
Thank you for your collaboration, I think this widget could interest a lot of Osmand users.
Best regards

**Comment by EugeneZmeuk** [comment]
https://www.youtube.com/watch?v=MwX5mbbiCFk&ab_channel=DavidLemereis
I think Drive mode app is for this use-case.

**Comment by fifilrv8** [comment]
Hello, that's right, this app seems very interesting to me for my trips to the Roadbook. Thank you very much for the info
best regards
